modular_skyrat/modules/autotransfer/code/transfer_vote.dm

```diff
@@ -16,16 +16,15 @@
 	return TRUE
 
 /datum/vote/transfer_vote/is_config_enabled()
-	return CONFIG_GET(flag/allow_vote_restart)
+	return CONFIG_GET(flag/autotransfer)
 
 /datum/vote/transfer_vote/can_be_initiated(mob/by_who, forced)
 	. = ..()
 	if(!.)
 		return FALSE
 
-	if(!forced && !CONFIG_GET(flag/allow_vote_restart))
```

Comment:
Won't this mean it won't respect the config anymore?

Reply:
Fixed